include/databricks_helper.py

```diff
@@ -0,0 +1,8 @@
+from airflow.providers.databricks.hooks.databricks import DatabricksHook
+
+def get_notebook_output(databricks_run_id: str):
+    databricks_hook = DatabricksHook()
+
+    response = databricks_hook._do_api_call(("GET", f'api/2.0/jobs/runs/get-output?run_id={databricks_run_id}'), {})
```

Comment:
Just curious if the [DatabricksHook.get_run_output method](https://github.com/apache/airflow/blob/16adc035b1ecdf533f44fbb3e32bea972127bb71/airflow/providers/databricks/hooks/databricks.py#L268) is suitable here? It feels a _little_ strange using a "private" method from the hook. But hey, if it's not good to use, smells like an OSS PR enhancement.

Reply:
Agree, I did not do my diligence to check what functions are available.